src/pages/api/updateprofilepic.ts

```diff
@@ -0,0 +1,21 @@
+import { NextApiRequest, NextApiResponse } from 'next';
+import { userAccounts } from '../../server/models';
+import { getData, updateProfilePic } from '../../server/db';
+
+export default async function UpdateProfilePic(
```

Comment:
The name/route for this is not consistent with common REST naming conventions, in which resources are nouns, and the verbs are the HTTP methods. I would have called this route `/api/profilepic` and to "update" it I would use `PUT` or `PATCH`.

Or, since what you're really updating is the account data model, I would have designed the API such that a `PATCH` to `/api/useraccount` could include a reference to the profile picture, so that this API route would be entirely redundant.